contrib/drivers/hpe/nimble/model.go

```diff
@@ -0,0 +1,468 @@
+// Copyright 2019 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nimble
+
+type AuthRespBody struct {
+	Data AuthRespData `json:"data"`
+}
+
+type AuthRespData struct {
+	AppName      string `json:"app_name"`
+	CreationTime int    `json:"creation_time"`
+	Id           int    `json:"id"`
+	LastModified int    `json:"last_modified"`
+	SessionToken string `json:"session_token"`
+	SourceIp     string `json:"source_ip"`
+	UserName     string `json:"username"`
+}
+
+type ClinetErrors struct {
+	Errs []error
+}
+
+type ArrayInnerErrorBody struct {
+	Errs []ArrayInnerErrorResp `json:"messages"`
+}
+
+type ArrayInnerErrorResp struct {
+	Code     string `json:"code"`
+	Severity string `json:"severity"`
+	Text     string `json:"text"`
+}
+
+type StoragePoolsRespBody struct {
+	StartRow  int                   `json:"startRow"`
+	EndRow    int                   `json:"endRow"`
+	TotalRows int                   `json:"totalRows"`
+	Data      []StoragePoolRespData `json:"data"`
+}
+
+type StoragePoolRespData struct {
+	Id            string      `json:"id"`
+	Name          string      `json:"name"`
+	Description   string      `json:"description"`
+	TotalCapacity int64       `json:"capacity"`
+	FreeCapacity  int64       `json:"free_space"`
+	ArrayList     []ArrayList `json:"array_list"`
+	Endpoint      string
+	Token         string
+}
+
+type ArrayList struct {
+	ArrayId   string `json:"array_list"`
```

Comment:
I'm confused why the label here is `array_list`?

Reply:
Thanks, let me check.

Reply:
This is invalid label. I fixed it.